web/instances/instances.go

```diff
@@ -360,6 +360,22 @@ func createSessionCode(c echo.Context) error {
 	})
 }
 
+func cleanSessions(c echo.Context) error {
+	domain := c.Param("domain")
+	inst, err := lifecycle.GetInstance(domain)
+	if err != nil {
+		return err
+	}
+
+	if err := couchdb.DeleteDB(inst, consts.Sessions); err != nil {
+		return err
```

Comment:
If there is no sessions database, then the sessionsLogin database will not be delete.
Maybe you can use a cumulative error handler and only return after atempting to delete both databases? or ignore 404 errors?